apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1/RetryTiming.cs

```diff
@@ -0,0 +1,109 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License").
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at 
+//
+// https://www.apache.org/licenses/LICENSE-2.0 
+//
+// Unless required by applicable law or agreed to in writing, software 
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and 
+// limitations under the License.
+
+using System;
+using System.Threading.Tasks;
+
+namespace Google.Cloud.Storage.V1;
+
+/// <summary>
+/// Options to control the delays between attempts when retrying failed API requests. These are immutable.
+/// </summary>
+public sealed class RetryTiming
+{
+    /// <summary>
+    /// The backoff time between the first attempt and the first retry.
+    /// </summary>
+    public TimeSpan InitialBackoff { get; }
+
+    /// <summary>
+    /// Maximum backoff time between retries.
+    /// </summary>
+    public TimeSpan MaxBackoff { get; }
+
+    /// <summary>
+    /// The multiplier to apply to the backoff on each iteration.
```

Comment:
```suggestion
    /// The multiplier to apply to the previous backoff on each iteration, leading to exponential backoff.
```

(Just to make it clear.)